### PR TITLE
refactor(editor): Migrate mouseSelect & deviceSupportHelpers mixins to composables (no-changelog)

### PR DESCRIPTION
**Pull request by OlegIvaniv**
- Migrate `mouseSelect` and `deviceSupportHelpers` to composable modules
- Move `newNodeInsertPosition` and `lastSelectedConnection` to the `canvasStore` so it could be used from the `useCanvasMouseSelect`.

### CODE REVIEW

File: packages/editor-ui/src/views/NodeView.vue [hunk 1]
```diff
@@ -1682,7 +1678,8 @@ export default mixins(
 							// If adding more than one node, offset the X position
 							mousePosition[0] -
 								NodeViewUtils.NODE_SIZE / 2 +
-								NodeViewUtils.NODE_SIZE * (index * 2 + NodeViewUtils.GRID_SIZE),
+								NodeViewUtils.NODE_SIZE * index * 2 +
+								NodeViewUtils.GRID_SIZE,
```
Comment (mutdmour) [inline packages/editor-ui/src/views/NodeView.vue:1682]: was this intentional change?

**Review by mutdmour — APPROVED** [review]
LGTM

File: packages/editor-ui/src/views/NodeView.vue (hunk 1 above)
Comment (OlegIvaniv) [reply packages/editor-ui/src/views/NodeView.vue:1682]: Yes. It fixes the node insert position when dropping

**Comment by janober** [comment]
Got released with [`n8n@0.222.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.222.0)

**Pull request merged**